DSharpPlus.Commands/Processors/TextCommands/TextCommandProcessor.cs

```diff
@@ -312,19 +312,19 @@ public bool TryGetCommand(string commandText, ulong guildId, out int index, [Not
             }
 
             // If there was no space found after the subcommand, break
-            nextIndex = commandText.IndexOf(' ', nextIndex);
+            nextIndex = commandText.IndexOf(' ', nextIndex + 1);
             if (nextIndex == -1)
             {
                 // No more spaces. Search the rest of the string to see if there is a subcommand that matches.
                 nextIndex = commandText.Length;
             }
 
             // Resolve subcommands
-            string subcommandName = commandText[index..nextIndex];
+            string subCommandName = commandText[index..nextIndex];

```

Comment:
the capitalization here was correct

Reply:
It's inconsistent with the rest of the extension

Reply:
no, the extension is entirely inconsistent; https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus.Commands/Trees/Command.cs#L19 vs https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus.Commands/Trees/Command.cs#L25 for example

Reply:
I'll fix this in a separate PR, which fixes all of them